src/test/resources/sql/Answer_audit.sql

```diff
@@ -0,0 +1 @@
+create table Answer_audit(id int(11), sortOrder int(11), answerText varchar(255), action varchar(10),dbUser varchar(255), appUser varchar(255), createdTs timestamp)
```

Comment:
I did this that way because it's easier to maintain them separately than in 1 properties file. When any update in any file will be necessary I can generate and replace it in no time while updating properties file is not so easy to automate.
And yes those files represent expected results but also they are expected changes in the database to be picked and processed by configurator. 
Answer_audit.sql and User_audit.sql are file with respective table structure representing the state of the database, the rest of the files (having prefixes/suffixes something like: _alter, create_, drop_ ) contain the expected modification statements which are compared with those generated by unit test.